doc/mgr/ansible.rst

```diff
@@ -120,3 +120,56 @@ Example::
     sda (hdd, 53687091200b)
     sdb (hdd, 53687091200b)
     sdc (hdd, 53687091200b)
+
+**Create OSD:**
+
+Create a osd in one or more hosts using a drive group specification:
+
+orchestrator osd create {<osd_host>} {<json>} {<file>}
+
+Where:
+
+{<osd_host>}: Optional parameter to create OSD's only in this host
+{<json>}: String with json format used to specify 'lvm_volumes' parameter
+{<file>}: Name of the file that contains a json string to specify 'lvm_volumes' parameter
+
+The "lvm_volumes" parameter is a list with a dictionary that contains the Drive group attributes
+
+A Drive Group: Is set of parameters used to specify how to use the storage devices in one host in order to create one or more osd's.
```

Comment:
i'd suggest capitalize OSD. as it's the abbreviation of Object Storage Daemon.

Reply:
Done. Thx!